client/state/sharing/schema.js

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
Use case is, the connections subtree gets thrown out because the schemas don't match. FetchingConnections has those connections marked as completed or (false). Editor then fails to load fully.